File: scrimage-core/src/test/scala/com/sksamuel/scrimage/Issue29Test.scala
package com.sksamuel.scrimage

import org.scalatest.{FunSuite, OneInstancePerTest}

/** @author Stephen Samuel */
class Issue29Test extends FunSuite with OneInstancePerTest {

  test("image for issue 29 should load") {
    assert(ImmutableImage.fromStream(getClass.getResourceAsStream("/issue29.jpeg")) != null)
  }
}

File: scrimage-core/src/test/kotlin/com/sksamuel/scrimage/core/Issue29Test.kt
package com.sksamuel.scrimage.core

import com.sksamuel.scrimage.ImmutableImage
import io.kotest.core.spec.style.FunSpec

class Issue29Test : FunSpec({

   test("image for issue 29 should load") {
      assert(ImmutableImage.fromStream(javaClass.getResourceAsStream("/issue29.jpeg")) != null)
   }
})